deck.py

```diff
@@ -0,0 +1,60 @@
+""""Sets up the Deck class for a network Poker game"""
+
+
+from card import Card
+from random import shuffle
+
+
+class Deck:
+    def __init__(self, deck_of_cards):
+        self.deck_of_cards = deck_of_cards
+
+
+    def __eq__(self, other):
+        """Return eq
+        >>> (
+        ... Deck() ==
+        ... Deck()
+        ... )
+        True
+        >>> (
+        ... Deck() ==
+        ... Deck(1)
+        ... )
+        False
+        """
+        return(
+            self.deck_of_cards == other.deck_of_cards
+        )
+
+
+    def __repr__(self):
+        """Return repr.
+        >>> repr(Deck([Card('H', '5'), Card('C', '2')]))
+        "Deck([Card('H', '5'), Card('C', '2')])"
+        """
+        return 'Deck({!r})'.format(
+            self.deck_of_cards
+        )
+
+
+def generate_deck(suits, ranks):
```

Comment:
It would make the main() cleaner if generate_deck() had suits and ranks internal, i.e., card.SUITS and card.RANKS instead of passing them to the function.  It is not likely we will be generating a deck from any other lists